plugins/bulk-action.yaml

```diff
@@ -21,11 +21,14 @@ spec:
   shortDescription: Do bulk actions on Kubernetes resources.
   caveats: |
     Usage:
-      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout) <command parameters>
+      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout|scale) <command parameters>
+
+    Help:
+      kubectl bulk-action help        
```

Comment:
Can you move this to the `description` tag? `caveats` should be reserved for dependencies.